exercise1.py

```diff
@@ -0,0 +1,18 @@
+# File     : program1.py
+# Brief    :A program for simple arithmetic operations like add, subtract, multiplication, division and etc
```

Comment:
Dont use # for file comments use doc string.
File name is exercise1 put inside you have give File     : program1.py